map_model/src/map.rs

```diff
@@ -579,23 +579,28 @@ impl Map {
 
     pub fn pathfind(&self, req: PathRequest) -> Result<Path> {
         assert!(!self.pathfinder_dirty);
-        self.pathfinder
+        let path = self
+            .pathfinder
             .pathfind(req.clone(), self)
-            .ok_or_else(|| anyhow!("can't fulfill {}", req))
+            .ok_or_else(|| anyhow!("can't fulfill {}", req))?;
+        path.to_v1(self)
```

Comment:
This is the overall approach to this PR -- internal to map_model, use v2 natively. When we give paths to the outside world, convert to v1.